nfp/preprocessing/features.py

```diff
@@ -85,3 +85,26 @@ def bond_features_v3(bond, flipped=False):
     return str((bond.GetBondType(), bond.GetIsConjugated(), bond.GetStereo(),
                 get_ring_size(bond), bond.GetEndAtom().GetSymbol(), start_atom,
                 end_atom))
+
+def atom_features_xtb(atom, n, json_data):
+    xtb_props = [
+        'mulliken charges', 'cm5 charges', 's proportion', 'p proportion',
+        'd proportion', 'FOD','FOD s proportion','FOD p proportion','FOD d proportion'
+    ]
```

Comment:
some of these must be floats, correct? I think the current implementation would treat a an atom with a charge of '0.001' as completely different from one with charge '0.002', which probably isn't what we want

Reply:
Done